controllers/grid/files/query/form/ManageQueryNoteFilesForm.inc.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/**
+ * @file controllers/grid/files/query/form/ManageQueryNoteFilesForm.inc.php
+ *
+ * Copyright (c) 2014-2015 Simon Fraser University Library
+ * Copyright (c) 2003-2015 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class ManageQueryNoteFilesForm
+ * @ingroup controllers_grid_files_query
+ *
+ * @brief Form to add files to the query files grid
+ */
+
+import('lib.pkp.controllers.grid.files.form.ManageSubmissionFilesForm');
+
+class ManageQueryNoteFilesForm extends ManageSubmissionFilesForm {
+	/** @var int Query ID */
+	var $_queryId;
+
+	/** @var int Note ID */
+	var $_noteId;
+
+	/** @var array Extra parameters to actions. */
+	var $_actionArgs;
+
+	/**
+	 * Constructor.
+	 * @param $submissionId int Submission ID.
+	 * @param $queryId int Query ID.
+	 * @param $noteId int Note ID.
+	 * @param $actionArgs array Optional list of extra request parameters.
+	 */
+	function ManageQueryNoteFilesForm($submissionId, $queryId, $noteId, $actionArgs = array()) {
+		parent::ManageSubmissionFilesForm($submissionId, 'controllers/grid/files/query/manageQueryNoteFiles.tpl');
+		$this->_queryId = $queryId;
+		$this->_noteId = $noteId;
+		$this->_actionArgs = $actionArgs;
+	}
+
+	/**
+	 * @copydoc Form::fetch()
+	 */
+	function fetch($request, $template = null, $display = false) {
+		$templateMgr = TemplateManager::getManager($request);
+		$templateMgr->assign(array(
+			'queryId' => $this->_queryId,
+			'noteId' => $this->_noteId,
+			'actionArgs' => $this->_actionArgs,
+		));
+		return parent::fetch($request, $template, $display);
+	}
+
+	/**
+	 * Save selection of query files
+	 * @param $args array
+	 * @param $request PKPRequest
+	 * @return array a list of all submission files marked as available to queries.
+	 */
+	function execute($args, $request, $stageSubmissionFiles) {
+		parent::execute($args, $request, $stageSubmissionFiles, SUBMISSION_FILE_QUERY);
+	}
+
+	/**
+	 * Determine if a file is already present in the stage.
+	 * @param $submissionFile SubmissionFile The submission file
+	 * @param $stageSubmissionFiles array The list of submission files in the stage.
+	 * @param $fileStage int FILE_STAGE_...
+	 */
+	protected function _fileExistsInStage($submissionFile, $stageSubmissionFiles, $fileStage) {
```

Comment:
extra underscore not necessary for protected methods, also in other places, I think.